nvflare/app_common/abstract/model_persistor.py

```diff
@@ -63,5 +102,6 @@ def get_model_inventory(self, fl_ctx: FLContext) -> {str: ModelDescriptor}:
         """
         pass
 
+    @abstractmethod
     def get_model(self, model_file, fl_ctx: FLContext) -> object:
         pass
```

Comment:
some existing ModelPersistor did not implement this method.

This is discussion: what do you think get_model should do? VS load_model

Reply:
Other components (like `PTFileModelLocator`) should be using `get()` now as this will call the filter with `process_post_load()`; `get()` is based on `model_file` while `load()` doesn't expect an input.

Reply:
Currently, load() is used initialize the global model, while `get()` is used to return the best and final global models for cross-site validation.

Reply:
You are right, `get_model_inventory()` and `get_model()` were not decorated as `@abstractmethod`. I kept it so that only `load_model()` and `save_model()` need to be implemented as before.